tutorials/kotlin.md

```diff
@@ -77,4 +74,4 @@ There are a number of [open source Kotlin projects](https://github.com/search?l=
 * [haskell-idea-plugin](https://github.com/atsky/haskell-idea-plugin)
 * [intellij-markdown](https://github.com/valich/intellij-markdown)
 * [IntelliJ-presentation-assistant](https://github.com/chashnikov/IntelliJ-presentation-assistant)
-* [leanpub-plugin](https://github.com/hhariri/leanpub-plugin)
\ No newline at end of file
+* [leanpub-plugin](https://github.com/hhariri/leanpub-plugin)
```

Comment:
Yep, this file lacked the trailing newline, and my editor automatically ensures that there is one. 

What is the convention for this repo? Judging by the surrounding files there isn't any...

Will remove this from the PR.